lib/presentation/widgets/man_widget.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class ManWidget extends StatelessWidget {
+  const ManWidget({Key? key}) : super(key: key);
+
+  static const manImagePath = "assets/images/man_login.png";
+  static const manHeight = 241.0;
```

Comment:
i would prefet to do picture size according to screen size. F.e, 1/4 of all screen heigth